frontend/components/WebsiteHeader.vue

```diff
@@ -33,8 +33,8 @@
           <div class="flex items-center space-x-3 lg:space-x-4 xl:space-x-6">
             <SelectorTheme />
             <SelectorLanguage />
-            <BtnLabeled :cta="true" label="get-in-touch" linkTo="/help/contact" fontSize="sm" class="hidden lg:block" />
-            <BtnLabeled :cta="true" label="get-in-touch" linkTo="/help/contact" fontSize="xs"
+            <BtnLabeled :cta="true" label="components.website-header.get-in-touch" linkTo="/help/contact" fontSize="sm" class="hidden lg:block" />
```

Comment:
Would it make sense for something like this that's generally used on many pages to maybe shift it to `components.btn-labeled.label.get-in-touch`? So maybe if we have a component that's being used within another, the localized strings for it as "options" would be in that part of the JSON?

Reply:
The issue is that btnLabeled component can receive different labels. In this case, the text is from the WebsiteHeader. However, if another component instantiated the BtnLabeled, the path would be that component.

Reply:
Awake and checking this again 😊 But for things like `BtnLabeled` couldn't we just switch the last part of the key? So we could have:

- `components.btn-labeled.label.get-in-touch`
- `components.btn-labeled.label.learn-more`

And anywhere that that same text it used we could then apply it? As of now we'd need to potentially apply the same label that's technically on the same component, but it's just on different pages 🤔

Reply:
What I'm thinking is that the keys would be defined in relation to the lowest component on which they're found, or specifically in relation to the component in which they're applied. Yes we're currently within `WebsiteHeader` here, but the component is technically `BtnLabeled`.